imsim/stamp.py

```diff
@@ -16,8 +16,8 @@ class DiffractionPSF:
     azimuth: galsim.Angle
     altitude: galsim.Angle
     rotTelPos: galsim.Angle
+    brightness_threshold: float
```

Comment:
And because of this works with the MISSING below this parameter is being checked here.

Reply:
Why is this required now?  Can't we give this the usual default of 1.e5 still?

Reply:
One of the reasons for these changes was to make use of the per CCD values obtained from the camera the default behavior, both here and in the readout code.  Letting this be 1e5 for all CCDs would be defeat that aim.

Reply:
Thinking about this some more, it occurs to me that linking this brightness threshold for the diffraction spikes to the CCD saturation values really doesn't make any sense.  They are physically unrelated.  So we can just use set this to some reasonable value, and I can remove all of the awkward config-related code here that was intended to preserve that coupling.